theme/tutorial-sidebar.less

```diff
@@ -531,6 +532,7 @@
 .tabTutorial {
     .immersive-reader-button.ui.item {
         float: right;
+        margin-left: 1.5rem;
```

Comment:
it's not really that hard to implement at all if we wanted it, it would be something like

```css
.tutorial-step-content p:first-child {
    margin-top: 3rem;
}
```

that said, long titles would behave the same way if they get all the way across -- it feels like too much empty space to me to have a whole line dedicated to the imm reader . Maybe a left / buttom border on imm reader or something? I'll play around

Reply:
Right, I remember Jacqueline mentioning that, too. Okay, sounds good!

Reply:
hmm yeah how about with a small border, it doesn't feel bade to me and makes it pretty explicit it's a button separate


![imm-reader-margin-with-border](https://user-images.githubusercontent.com/5615930/192061915-b5794520-9141-4bcd-8f71-eff8b6b9b1d5.gif)

Reply:
Yes, I like the separation that creates, too!